src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -100,6 +100,8 @@ def execute(self, unexpanded_argv):
             self.session['command'] = expanded_arg.command
             try:
                 _validate_arguments(expanded_arg)
+            except CLIError:
```

Comment:
This is so that CLIErrors are not considered validation errors, which end up printing the usage statement.